components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/policy/Limit.java

```diff
@@ -18,33 +18,34 @@
 
 package org.wso2.carbon.apimgt.core.models.policy;
 
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
 /**
  * Contains {@link Limit} based attributes
```

Comment:
I don't see a usage of having a link to the same class. When someone reads this comment, they are already on the same class.